modules/openid_connect/app/views/openid_connect/providers/_azure_form.html.erb

```diff
@@ -5,13 +5,13 @@
                     'admin--openid-connect-providers-target': 'azureForm',
                   },
                   hidden: @provider.name.present? && @provider.name != 'azure' do %>
-    <div class="form--field">
+    <div class="form--field -required">
       <%= f.text_field :tenant, required: true, container_class: '-middle' %>
       <div class="form--field-instructions">
         <%= t('openid_connect.setting_instructions.azure_tenant_html') %>
       </div>
     </div>
-    <div class="form--field">
+    <div class="form--field -hidden">
       <%= f.check_box :use_graph_api, container_class: '-middle' %>
       <div class="form--field-instructions">
         <%= t('openid_connect.setting_instructions.azure_graph_api') %>
```

Comment:
I would suggest getting rid of the graph api option altogether but perhaps it's best to discuss this with Oliver as well.

Reply:
As long as we still keep the flag internally so that existing Azure customers using it without Graph API are not affected, I'm fine with removing it from the UI.

Reply:
So, @oliverguenther , are you generally satisfied with this simple fix, by hiding it from the form? There is one drawback though, you cannot update an old azure app, but instead have to recreate it. But this is what I wrote into the help text, too.